mirage/config.js

```diff
@@ -235,4 +236,15 @@ export default function() {
   membership microservice
   ---------------------------------------------------------------*/
   this.get(`${config.wnycMembershipAPI}/v1/orders/`, 'orders');
+  this.get(`${config.wnycMembershipAPI}/v1/emails/is-verified/`, {data: {is_verified: true}});
+  this.patch(`${config.wnycMembershipAPI}/v1/emails/:email_id/verify/`, (schema, request) => {
+    let params = JSON.parse(request.requestBody);
+    if (params &&
+        get(params, 'data.attributes.verification_token') &&
+        get(params, 'data.attributes.verification_token') !== "null") {
```

Comment:
Assuming `JSON.parse` always returns some value, could this just be written as

```javascript
if (get(params, 'data.attributes.verification_token'))
```
If `verfication_token` is `null` or empty it would be false, and the `get` function doesn't care if you try to call sub-attributes that don't exist (it just returns `undefined`)... so it should only return `true` if the token actually exists. (Right?)